fd/diff.go

```diff
@@ -12,19 +14,21 @@ import (
 	"github.com/gonum/floats"
 )
 
-// A Point is a stencil location in a difference method.
+// A Point is a stencil location in a finite difference formula.
```

Comment:
I don't like neither method nor scheme in this context, in my mind they are both too related to discretization of PDEs. I would prefer formula or kernel (or something else). I have a PR ready where I rename `Method` to `Formula`, I think that is a more accurate description. I will leave this as is and we can return to this later in that PR.